frontend/src/store.js

```diff
@@ -28,7 +28,7 @@ const init = {
   questionNumber: 0,
   classSize: 0,
   openExportModal: false,
-  currentQuestionInfo: JSON.parse(JSON.stringify(baseQuestionInfo)),
+  currentQuestionInfo: Object.assign({}, baseQuestionInfo),
```

Comment:
This is faster because it is not a deep copy. Please revert pack to JSON.parse and JSON.stringifiy